server/controllers/userAuthController.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable indent */
+import moment from 'moment';
+import uuid from 'uuid';
+import 'babel-polyfill';
+import jwt from 'jsonwebtoken';
+import db from '../db/tables';
+import dba from '../db/index';
+import userAuthHelper from '../validation/userAuth';
+
+/**
+ * 
+ */
+class userAuthControllerClass {   
+  /**
+     * Create A User
+     * @param {object} req
+     * @param {object} res
+     * @returns {object} user object
+     */
+    static async createUser(req, res) {
+  
+      if (!req.body.fullname || !req.body.email || !req.body.password || !req.body.confirmPassword) {
```

Comment:
Please abstract out your validation from this line to line 42 into something like the `helpers/validation/auth.js` file